### PR TITLE
Music 163 report fake 7 d plays 2 (Includes first pass of IaC)

### PR DESCRIPTION
**Pull request by mcbrownMedRhythms**
Update play reporting to publish fake data to 7D

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
```
Comment (mcbrownMedRhythms) [inline build.sh:2]: Add description

File: app.py [hunk 2]
```diff
@@ -51,43 +51,32 @@ def create_seven_digital_account(event):
...
-
-
-@app.on_sqs_message(queue=REPORT_PLAYS_TASK_QUEUE, name="hello_world")
-def report_plays_hello_world(event):
-    """
-    Lambda function for verifying db and internet connectivity.
-    """
-    # verify db via simple query
-    from chalicelib.db import Database
+    for record in event:
```
Comment (jcu721) [inline app.py:71]: could put expected event format in docstring. is it sending one report stream per song?

File: app.py [hunk 3]
```diff
@@ -51,43 +51,32 @@ def create_seven_digital_account(event):
...
-    play_log, transaction_detail = services.report_plays(
-        user_id=data["user_id"],
-        play_log_data=data["play_log_data"],
-    )
+    # play_log, transaction_detail = services.report_plays(
+    #     user_id=data["user_id"],
+    #     play_log_data=data["play_log_data"],
+    # )
 
     # prepare and return SQS response msg including transaction detail
```
Comment (jcu721) [inline app.py:69]: can delete commented code? Ending comment out of sync with return, could also add @returns value to docstring.

**Review by jcu721 — APPROVED** [review]
Infrastructure changes look good. Left some small comments around the report_plays code.

File: app.py (hunk 2 above)
Comment (mcbrownMedRhythms) [reply app.py:83]: I think this depends on the batch size configuration of the lambda. If we had a batch size larger than 1, I think we could get multiple records

**Pull request merged**